source/scripting/VehicleDoorCollection.cs

```diff
@@ -0,0 +1,45 @@
+using GTA.Native;
+using System;
+using System.Collections.Generic;
+
+namespace GTA
+{
+    public enum VehicleDoorIndex
+    {
+        FrontRightDoor = 1,
+        FrontLeftDoor = 0,
+        BackRightDoor = 3,
+        BackLeftDoor = 2,
+        Hood = 4,
+        Trunk
+    }
+
+    public sealed class VehicleDoorCollection
+    {
+        #region Fields
+        Vehicle _owner;
+        readonly Dictionary<int, VehicleDoor> _vehicleDoors = new Dictionary<int, VehicleDoor>();
```

Comment:
Why not `Dictionary<VehicleDoorIndex, VehicleDoor>`? Gets rid of the cast to int below that way.